--- a/testsuite/expect/regression.py       2019-11-01 19:56:10.848007246 +0000
+++ b/testsuite/expect/regression.py    2019-11-01 20:03:18.719706386 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 ############################################################################
 # Copyright (C) 2006 The Regents of the University of California.
 # Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
